### DISCUSSION TITLE
https certificate and key on v10, possible?

### DISCUSSION THREAD

**Discussion by psyntium** [sindresorhus/got#1721] [Q&A] [answered]
Hi,

I know that https certificate and key is an existing feature on got v11. Is there a way to replicate this feature on got v10 as my team is currently using got v10 in their codes and I can't just make a request for upgrade because of this small requirement.

Please advice.

**Comment by szmarczak** [comment, marked as answer]
It should be possible to pass the Node.js options in v10.

**Reply by psyntium** [reply]
just to clarify, I’m referring to got v10, not nodejs v10

**Reply by psyntium** [reply]
if you’re referring to got v10, can you share a sample?

**Reply by szmarczak** [reply]
https://github.com/sindresorhus/got/blob/v10.7.0/test/https.ts

**Reply by psyntium** [reply]
it works! thanks a bunch!

**Reply by psyntium** [reply]
sample code:
```javascript
got.post("https://google.com", {
   cert: fs.readFileSync("cert.pem", "utf8"),
   key: fs.readFileSync("key.pem", "utf8")
})
```
